app/locale/en.coffee

```diff
@@ -2691,8 +2691,9 @@
     ready_to_code: "I'm ready to code!"
 
   hoc_2018_completion:
-    congratulations: "Congratulations on completing Hour of Code!"
-    send: "Send your Code, Play, Share game to friends and family!"
+    congratulations: "Congratulations on completing" # {change}
```

Comment:
I think it would be better to have the congratulations text as a single message only but with html tags for making some of it bold like this: https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L12, instead of dividing it into 3 parts.